Documentation/compatibility/SignedXml.GetPublicKey-returns-RSACng-on-net462.md

```diff
@@ -0,0 +1,38 @@
+## SignedXml.GetPublicKey returns RSACng on net462 (or lightup) without retargeting change
+
+### Scope
+Edge
+
+### Version Introduced
+4.6.2
+
+### Version Reverted
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+We let an object return type change in net462 without quirking it. This change adds a quirk to use the old method of getting the return value (certificate.PublicKey.Key) instead of the new method (certificate.GetAnyPublicKey)
+
+The concrete type of the object returned by the SignedXml.GetPublicKey method changed (without a quirk) from a CryptoServiceProvider implementation to a Cng implementation in 4.6.2. This is because the implementation changed from usingcertificate.PublicKey.Key to using the internal certificate.GetAnyPublicKey in 4.6.2 which forwards e.g. to RSACertificateExtensions.GetRSAPublicKey.
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
```

Comment:
We need to be a bit more clear about the change here. I take it that this was a runtime change affecting all apps regardless of target framework, but that starting in 4.7.1 a quirk lets you restore the behavior from 4.6.1 and earlier. My suggestions below assume that's the case.

Reply:
That is correct, yes.